crates/autopilot/Cargo.toml

```diff
@@ -15,7 +15,7 @@ name = "autopilot"
 path = "src/main.rs"
 
 [dependencies]
-alloy = { workspace = true }
+alloy = { workspace = true, features = ["reqwest-default-tls"]}
```

Comment:
Shouldn't this rather be added to the `ethrpc` crate? For example I expect the same issue to be in the `orderbook`, `solvers`, and `refunder` binaries.